pages/views/styles/sidebar.less

```diff
@@ -0,0 +1,35 @@
+.sidebar {
+  display: none;
+  background-color: @dark-bg;
+  box-shadow: 4px 8px 8px 0 rgba(0, 0, 0, .5);
+  position: fixed;
+  top: @header-height;
+  z-index: 10;
+  width: 200px;
+  max-width: 200px;
+  /* stylelint-disable */
```

Comment:
Why does the linter consider messy this line, possibly it make sense to update stylelint config

Reply:
I just forgot to remove unnecessary comments :) thank you!